packages/datadog-ci/README.md

```diff
@@ -44,90 +44,90 @@ The following values are available for each `<command>` and (optionally) `<subco
 See each command's linked README for more details, or click on [📚](https://docs.datadoghq.com/) to see the related documentation page.
 
 #### `aas`
-- `instrument`: Apply Datadog instrumentation to the given [Azure App Services](packages/datadog-ci/src/commands/aas). [📚](https://docs.datadoghq.com/serverless/azure_app_services/)
-- `uninstrument`: Revert Datadog instrumentation from the given [Azure App Services](packages/datadog-ci/src/commands/aas).
+- `instrument`: Apply Datadog instrumentation to the given [Azure App Services](/packages/plugin-aas). [📚](https://docs.datadoghq.com/serverless/azure_app_services/)
+- `uninstrument`: Revert Datadog instrumentation from the given [Azure App Services](/packages/plugin-aas).
```

Comment:
Few updates in this file:
1. Many references to plugins were outdated
2. Many references were pointing to /src, but the readme is now in the main folder
3. Adding a leading slash because now that this file is not in the root, links only work if accessed through the main folder and not in its current location